pkg/controller/artifacts/README.md

```diff
@@ -0,0 +1,22 @@
+Network CRD & related controllers
+=================================
+
+## Network CRD
+
+Network CRD is the custom API type to provide hard network isolation. Please see [multi-tenancy network design proposal](../../../docs/design-proposals/multi-tenancy/multi-tenancy-network.md). 
+
+The CRD yaml file, crd-network.yaml, __MUST__ be applied __BEFORE__ any controller runs, in cluster provision process.
```

Comment:
This is a little unusual for a CRD object.  Besides this, there are some other potential problems of the CRD approach are: 1) Native objects like pod/service will reference this CRD object. 2) this CRD needs to be shared among all tenants, but our multi-tenancy doesn't support sharing CRD cross tenants yet.

In general, I still don't know which approach is better -- CRD or a native object.  Let's compare these the pros and cons and choose one PR :)

Reply:
We had a meeting to discuss this. The decision is that: 1)  we will go with the CRD approach. 2) Multi-tenancy will provide CRD sharing feature. 3) Our network operator needs to be tenancy-aware and it will only be deployed in system space.

Reply:
per offline discussion, CRD approach was agreed upon.